### ISSUE TITLE
Player will retrigger itself

### ISSUE THREAD

**Issue by oathzed** [rhenwinch/Flixclusive#79]
### Steps to reproduce

1. Play a movie you haven't played before
2. Wait till the pop up that comes up saying you can play before the whole scrape is complete, it will launch the player
3. 3. Back out of the player, now wait on that movie screen
4. 4. I believe when the full search is complete, it retriggers itself
5. 5. The player will open again.

### Expected behavior

For it to not retrigger itself if you back out

### Actual behavior

It retriggers itself after you've backed out

### Flixclusive version

Latest always 1.4.0 pre

### Android version

Android 13

### Device

Pixel 7

### Other details

https://github.com/rhenwinch/Flixclusive/assets/52545913/ab9f6ac0-d132-424b-99dd-7325c7fa7924

Video is 30 seconds, wait till the end

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the **[latest version](https://github.com/rhenwinch/Flixclusive/releases/latest)**.
